packages/expo-flutter-adapter/ios/Classes/ExpoFlutterAdapterPlugin.m

```diff
@@ -106,41 +107,41 @@ - (void)applicationDidEnterBackground:(nonnull UIApplication *)application {
 
 @end
 
-#pragma mark EXCore Logging Blocks
-
-extern void EXLogInfo(NSString *format, ...) {
-    va_list args;
-    va_start(args, format);
-    NSString *message = [[NSString alloc] initWithFormat:format arguments:args];
-    va_end(args);
-    NSLog(@"%@", message);
-}
-
-extern void EXLogWarn(NSString *format, ...) {
-    va_list args;
-    va_start(args, format);
-    NSString *message = [[NSString alloc] initWithFormat:format arguments:args];
-    va_end(args);
-    NSLog(@"%@", message);
-}
-
-extern void EXLogError(NSString *format, ...) {
-    va_list args;
-    va_start(args, format);
-    NSString *message = [[NSString alloc] initWithFormat:format arguments:args];
-    va_end(args);
-    NSLog(@"%@", message);
-}
-
-extern void EXFatal(NSError *error) {
-    NSLog(@"%@",[error description]);
-}
-
-extern NSError * EXErrorWithMessage(NSString *message) {
-    NSLog(@"%@", message);
-    return nil;
-}
-
-extern UIApplication *EXSharedApplication() {
-    return nil;
-}
+#pragma mark UMCore Logging Blocks
+
+//extern void UMLogInfo(NSString *format, ...) {
```

Comment:
True, this should be removable! The definition now lives in `UMCore`. 

https://github.com/expo/expo/blob/master/packages/%40unimodules/core/ios/UMCore/Services/UMLogManager.m#L50